### PR TITLE
Add OMERO.iviewer page

### PR DESCRIPTION
**Pull request by hflynn**
Adds an OMERO.iviewer page using the Figure page as a template.

Still needs an OMERO.iviewer 'logo' (coloured text) which Liza will provide next week - I've left the figure one in place for now as a place holder (and possibly a new background image for header if Liza fancies making one) but is ready for content review.

It would be good to have a round number of features for the display so suggestions of which one to ditch or for another 3 to add would be good.

Video and release links can be updated when 1.0.0 is out.

### CODE REVIEW

**Comment by hflynn** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/omero/iviewer/

File: omero/iviewer/index.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                <h5>Open multi-selected images</h5>
+                <p>
+                    Support for multiple side-by-side viewers and the ability
+                    to synchronize settings across datasets.
+                </p>
+            </div>
+            <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-edit fa-2x"></i>
+                <h5>Draw and customize ROIs</h5>
+                <p>Draw shapes and select colours and line intensity.</p>
```
Comment (mtbc) [inline omero/iviewer/index.html]: colors

File: _includes/cta-omero-iviewer.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!-- begin CTA -->
+<hr class="whitespace">
+<header class="marketing-hero marketing-hero-omero-figure">
+    <div class="row">
+        <div class="small-12 columns bg-image-text-container-cta text-center">
+            <h2>Power of the web</h2>
+            <p>
+                Using OpenLayers technology, get all the functionality of our
+                desktop client in the next generation web-based image viewer
```
Comment (mtbc) [inline _includes/cta-omero-iviewer.html]: Import is a popular functionality of our desktop client.

Comment (hflynn) [reply _includes/cta-omero-iviewer.html]: I'll add 'visualization and annotation'

File: omero/iviewer/index.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    <header class="marketing-hero marketing-hero-omero-figure">
+        <div class="row text-center bg-image-text-container-short">
+            <div class="small-12 columns">
+                <div class="medium-6 columns">
+                    <div class="responsive-embed">
+                        <iframe src="//www.youtube.com/embed/FQD540TTwFg?rel=0" frameborder="0" allowfullscreen></iframe>
+                    </div>
+                </div>
+                <div class="medium-5 columns">
+                    <h1 class="hero-main-header"><img src="{{ site.baseurl }}/img/logos/omero-figure.svg" alt="OMERO.figure logo" /></h1>
```
Comment (will-moore) [inline omero/iviewer/index.html]: Needs in iviewer logo, not figure.

File: omero/iviewer/index.html [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+                <i class="icon-fa-blue fa fa-signal fa-2x"></i>
+                <h5>View histograms</h5>
+                <p>Full support for histograms of image intensity.</p>
+            </div>
+            <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-calendar fa-2x"></i>
+                <h5>View metadata</h5>
+                <p>
+                    Both acquisition and import date is available in the
+                    'info' tab.
```
Comment (will-moore) [inline omero/iviewer/index.html]: I think probably the image dimensions and pixel sizes are the most useful metadata. Maybe:
"Image dimensions, pixel sizes and acquisition date are available in the 'info' tab".

File: omero/iviewer/index.html [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+                </p>
+            </div>
+            <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-copy fa-2x"></i>
+                <h5>Open multi-selected images</h5>
+                <p>
+                    Support for multiple side-by-side viewers and the ability
+                    to synchronize settings across datasets.
```
Comment (will-moore) [inline omero/iviewer/index.html]: I think "datasets" could be confusing. Maybe just "...synchronize settings between them".

File: omero/iviewer/index.html (hunk 3 above)
Comment (hflynn) [reply omero/iviewer/index.html]: Yep, i left that in as a placeholder, as I mentioned in the description - Liza hasn't got time to do one until next week

**Comment by hflynn** [comment]
Can we come up with 3 more features to get a nice round number for the layout?

**Comment by hflynn** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/omero/iviewer/ updated

**Comment by will-moore** [comment]
I wonder if we want to make more of the features that are not supported in the old web viewer. 
A couple that are currently not mentioned are:
 - Inspect Pixel intensity by mousing over the image
 - Click on Dataset thumbnails to view images

Other new features which are mentioned could maybe be listed ahead of 'old' features?:
 - Smooth zoom on tiled images and rotation
 - Open and sync multiple viewers
 - Draw, Copy, Paste ROIs
 - Export ROI pixel intensity as CSV

File: omero/iviewer/index.html [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+            </div>
+            <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-paste fa-2x"></i>
+                <h5>Copy and paste ROIs</h5>
+                <p>Duplicate ROIs across multiple images.</p>
+            </div>
+           <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-share fa-2x"></i>
+                <h5>Export ROI statistics</h5>
+                <p>Export data as .csv files.</p>
```
Comment (will-moore) [inline omero/iviewer/index.html]: "Export ROI pixel intensities as .csv files" gives a bit more info

**Comment by hflynn** [comment]
Thanks @will-moore - https://snoopycrimecop.github.io/www.openmicroscopy.org/omero/iviewer/ looking better now?

**Comment by lunson** [comment]
Some visual changes here: https://github.com/lunson/www.openmicroscopy.org/tree/style-iviewer

Also, I refactored the layout/include files so that making a landing page like this in the future becomes easier (hopefully).

**Comment by hflynn** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/omero/iviewer/ updated and ready for final review. Thanks for those updates @lunson - looks great :)

File: omero/iviewer/index.html [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+                <h5>View metadata</h5>
+                <p>
+                    Image dimensions, pixel sizes, and acquisition and import
+                    dates are available in the 'info' tab.
+                </p>
+            </div>
+            <div class="figure-feature column">
+                <i class="icon-fa-blue fa fa-share-square fa-2x"></i>
+                <h5>Export images</h5>
+                <p>Export viewport images as .png files.</p>
```
Comment (manics) [inline omero/iviewer/index.html]: Optional: I think `PNG files` looks better than `.png files`.

**Comment by hflynn** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/omero/iviewer/ and https://snoopycrimecop.github.io/www.openmicroscopy.org/omero/features/view/ updated

**Comment by hflynn** [comment]
@jburel can we merge this and deploy it and then update the video and release announcement link on the same PR as the release announcement for 1.0.0?

**Comment by jburel** [comment]
@hflynn looks good. let's deploy. We can do that tomorrow morning.

**Pull request merged**